node/runtime/src/system.rs

```diff
@@ -390,6 +392,7 @@ mod tests {
         assert_eq!(apply_results[1].tx_result[0].status, TransactionStatus::Failed);
         assert_eq!(apply_results[2].tx_result[0].status, TransactionStatus::Completed);
         // assert_eq!(apply_result.new_receipts.len(), 0);
+        let viewer = StateDbViewer::new(state_db.clone());
```

Comment:
Out of scope for this, just a question: If runtime doesn't have state should state viewer have it?

Reply:
Yeah I was going to do that in a separate PR. Do you think I should do it in this one?